Common/MVKLogging.h

```diff
@@ -168,6 +185,7 @@ extern "C" {
 #endif
 
 #define MVKLogErrorImpl(fmt, ...)		reportMessage(ASL_LEVEL_ERR, fmt, ##__VA_ARGS__)
+#define MVKLogWarningImpl(fmt, ...)     reportMessage(ASL_LEVEL_WARNING, fmt, ##__VA_ARGS__)
```

Comment:
> For now, if we're happy not logging warnings when configured for error only

Yeah, I think it would be ok if we don't log the warnings when configured for error only, because Vulkan warnings actually indicates not immediately harmful problems. I hope I haven't added unnecessary problems to you with refactoring the logging system :sweat_smile:.